src/pages/Landing/index.tsx

```diff
@@ -150,9 +173,15 @@ const SubText = styled.div`
   }
 `
 
-const SubTextContainer = styled.div`
+const SubTextContainer = styled.div<{ $visible: boolean }>`
   display: flex;
   justify-content: center;
+
+  ${({ $visible }) =>
```

Comment:
should there be CSS which hides this component when $visible is false?

Reply:
nothing gets displayed anyway but i shall just add one since it won't hurt